<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Tournament_Entry">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Tournament_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tournament Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Tournament_Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Tournament</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Format_ComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>T20I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ODI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Test</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Format_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Format:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Category_ComboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mens</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Womens</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Category_Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f00;</string>
   </property>
   <property name="text">
    <string>Please Specify Tournament Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
